saleor/dashboard/order/views.py

```diff
@@ -15,10 +15,16 @@
 from ...core.exceptions import InsufficientStock
 from ...core.utils import get_paginator_items
 from ...core.utils.taxes import get_taxes_for_address
-from ...order import OrderEvents, OrderEventsEmails, OrderStatus
+from ...order import OrderStatus
 from ...order.emails import (
-    send_fulfillment_confirmation, send_fulfillment_update,
+    send_fulfillment_confirmation_to_customer, send_fulfillment_update,
     send_order_confirmation)
+from ...order.events import (
```

Comment:
We could also do `from ...order import events` to not have to worry about formatting and ordering of these all functions here.